packages/components/src/tabs/tests/chromatic/TabsVertical.chroma.jsx

```diff
@@ -0,0 +1,181 @@
+import { NotificationMajorIcon } from "@components/icons";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong